introduction.mdx

```diff
@@ -8,12 +8,11 @@ import EdgeeSdk from '/snippets/edgee-sdk.mdx';
 
 <EdgeeSdk />
 
-Edgee is an **edge platform** designed to transform how workloads and data are
+Edgee is an **edge component platform** designed to transform how workloads and data are
 processed across the internet.
 
-Edgee empowers developers to build and deploy **edge-oriented applications**.
-This approach to application development harnesses the potential of edge computing,
-enabling workloads to run closer to users than ever before.
+Edgee empowers developers to use, build and deploy **wasm components** operating at the edge.
```

Comment:
Let's try to use either "Wasm" or "WebAssembly".

Reply:
I think we need both, especially because of the SEO

Reply:
Yeah what I meant is to avoid lowercased "wasm" :) it should be "Wasm"